packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -14,6 +14,7 @@ import { PlayBilling } from './google-play/play-billing';
 import { SubscriptionPurchase } from './google-play/subscription-purchase';
 import { StripeHelper } from './stripe';
 import error from '../error';
+import { PurchaseQueryError } from './google-play/types';
```

Comment:
What are some possible root causes for this error? Is there anything in our control?

Reply:
Nope, this one is purely from Firestore.